strr-web/pages/registration-details/[id]/index.vue

```diff
@@ -33,49 +33,57 @@
         <h2 class="font-bold mb-6 mobile:mx-2 text-xl">
           {{ tApplicationDetails('unitInfo') }}
         </h2>
-        <div class="bg-white py-[22px] px-[30px] mobile:px-2">
-          <div class="flex flex-row justify-between w-full mobile:flex-col desktop:mb-6">
+        <div class="bg-white py-[22px] px-[30px] mobile:px-5" data-test-id="rental-unit-info">
+          <div class="grid grid-cols-1 tablet:grid-cols-2 desktop:grid-cols-3 gap-6 desktop:mb-6">
             <BcrosFormSectionReviewItem
               :title="tApplicationDetails('nickname')"
               :content="application?.unitAddress.nickname || '-'"
+              data-test-id="unit-nickname"
               class="break-all"
             />
             <BcrosFormSectionReviewItem
               :title="tApplicationDetails('ownership')"
               :content="getOwnershipTypeDisplay(application?.unitDetails.ownershipType, tApplicationDetails)"
+              data-test-id="ownership-type"
             />
-            <BcrosFormSectionReviewItem :title="tApplicationDetails('propertyType')">
-              <p>
-                {{ application?.unitDetails.propertyType
-                  ? tPropertyForm(propertyTypeMap[application?.unitDetails.propertyType as keyof PropertyTypeMapI])
-                  : '-'
-                }}
-              </p>
-            </BcrosFormSectionReviewItem>
-          </div>
-          <div class="flex flex-row justify-between w-full mobile:flex-col">
             <BcrosFormSectionReviewItem
               :title="tApplicationDetails('rentalUnitSpaceType')"
-              :content="tApplicationDetails(application?.unitDetails.rentalUnitSpaceType) || '-'"
+              :content="tApplicationDetails(application?.unitDetails.rentalUnitSpaceType)|| '-'"
```

Comment:
if `rentalUnitSpaceType` is empty this will fail

Reply:
okay addressed this